.swiftformat

```diff
@@ -54,7 +54,7 @@
 --patternlet hoist
 --ranges spaced
 --redundanttype infer-locals-only
---self remove
+--self insert
```

Comment:
Maybe a controversial change but I think it makes sense.

Swift allows you to omit `self` when using local properties or functions. This is great in Xcode where you have syntax highlighting so you can easily differentiate between local properties/functions or global ones.

This context is completely lost in GitHub where code review happens. To make it easier for reviewers I would propose to enforce `self` usage.

Whats your take on this, is this fine or should I revert this?

Reply:
I don't have strong feelings on this one. The default seems to be to remove self. If in doubt, I'd say, let's stick with the defaults unless we have a good reason not to follow defaults. So I am leaning towards `--self remove` or not to mention the parameter at all and fall back to the default?

Reply:
My reasoning here was to improve readability when reviewing in GitHub. Is this not a strong enough reasoning for you?